### ISSUE TITLE
/bin/bash: ./darknet: No such file or directory

### ISSUE THREAD

**Issue by valeriylo** [AlexeyAB/darknet#7060]
Yesterday tried to compile your colab code and everything was fine. Today I got this error in object detector cell

`# run darknet detection on test images`
`!./darknet detector test cfg/coco.data cfg/yolov4.cfg yolov4.weights data/person.jpg`

Reproduced this issues on several machines. Here's local storage:
![Снимок экрана от 2020-12-01 13-22-10](https://user-images.githubusercontent.com/45401765/100728253-611a5400-33d8-11eb-8817-6dde427d28a7.png)

**Comment by newbie2X** [comment]
Same Error for me.

**Comment by EZL1190** [comment]
Same for me, worked yesterday tho...

**Comment by Vikchoc** [comment]
Same Problem for me，worked 12h ago.Caused by the upgrade for RTX3090?

**Comment by EZL1190** [comment]
yeah, i made it work again by changing [this](https://github.com/AlexeyAB/darknet/commit/e87042a49a9e9196482f8bad6374b5e4f6ad364b) commit back to how it was before

**Comment by newbie2X** [comment]
@EZL1190 That worked. Thx man.

**Comment by Vikchoc** [comment]
simply use the old version can solve:
!git clone https://github.com/AlexeyAB/darknet
%cd darknet/
!git checkout 80e1da2
